wca-regulations.md

```diff
@@ -515,6 +519,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 - F2) The judge places the scrambled puzzle onto the mat in a standing position.
     - F2a) The organization team may enforce using a stand for the puzzle to prevent it from falling before the start of the attempt. If such stands are used, the organization team should make an announcement before the round starts.
         - F2a1) The judge must remove the stand from the mat immediately after the competitor removes the puzzle from it.
+    - F2b) If, during the transportation phase or while placing the puzzle cover on the table, one or more of the pins have their state affected, the competitor may proceed with the attempt and, after finishing the solve, alongside with the judge, explain the situation to the WCA Delegate. If the WCA Delegate beleives that the competitor did not gain an unfair advantage, they may allow the result to stand under their discretion.
```

Comment:
I think it would be beneficial for Delegates and competitors to see the Clarification F1a1+++ in this regulation (That an extra should be given if the competitor sees the puzzle). The way the regulation is currently worded, it may cause some confusion and judges / competitors may allow the attempt to continue and proceed even if the competitor saw the physical puzzle.

It would also be beneficial to define the case where the puzzle falls over during inspection (ie after the cover is lifted) as some may try to apply this regulation to that case, which would not be correct.

Reply:
This last sentence of this regulation seems to vary a bit from other regulations that involve delegate discretion. Perhaps something like the following would fit better?

The result may stand if the competitor did not gain an unfair advantage, at the discretion of the WCA Delegate.